qemu/tests/cfg/slof_hugepage.cfg

```diff
@@ -0,0 +1,38 @@
+- slof_hugepage: install setup image_copy unattended_install.cdrom
+    virt_test_type = qemu
+    type = slof_hugepage
+    restart_vm = yes
+    kill_vm_on_error = yes
+    login_timeout = 240
+    start_vm = no
+    slots_mem = 4
+    mem = 2048
+    maxmem_mem = 32G
+    pre_command = "echo 3 > /proc/sys/vm/drop_caches"
```

Comment:
It is better to flush dirty pages first  before dropping caches:
`pre_command = "sync & echo 3 > /proc/sys/vm/drop_caches"`

Reply:
Please replace commands with functions in `avocado.utils.memory`.

Reply:
For the 2nd comment, what's the difference between with both of them?